subsys/bluetooth/host/direction.c

```diff
@@ -593,6 +610,78 @@ int hci_df_prepare_connection_iq_report(struct net_buf *buf,
 }
 #endif /* CONFIG_BT_DF_CONNECTION_CTE_RX */
 
+#if defined(CONFIG_BT_DF_CONNECTION_CTE_REQ)
+static bool validate_cte_req_params(const struct bt_conn *conn, uint8_t cte_type,
```

Comment:
I'm not sure the name of this function is quite ideal. Since it returns bool the name should IMO be a boolean assertion (now the form of the verb makes it a request). I'd just follow similar naming as we've used elsewhere for parameter validation.

Reply:
Agreed. 

Suggest to follow the naming convention used by e.g. 
```
static bool valid_adv_ext_param(const struct bt_le_adv_param *param)
```
thus renaming this to 
```
static bool valid_cte_req_params(const struct bt_le_adv_param *param)
```

Reply:
Yeah, that's fine with me

Reply:
Changed according to @Thalley proposal.
I've also aligned other functions names `validate_**` in additional commit.